Docs/Configuration.tex

```diff
@@ -2626,31 +2632,34 @@ \subsection{Scheme Properties}\label{kernelpropsscheme}
 \item
   \texttt{KernelArch}\\
   \textbf{Type}: \texttt{plist\ string}\\
-  \textbf{Failsafe}: \texttt{Auto}\\
-  \textbf{Description}: Prefer specified kernel architecture (\texttt{Auto}, \texttt{i386},
+  \textbf{Failsafe}: \texttt{Auto} (Choose the preferred architecture automatically)\\
+  \textbf{Description}: Prefer specified kernel architecture (\texttt{i386},
   \texttt{i386-user32}, \texttt{x86\_64}) when available.
 
-  On macOS 10.7 and earlier XNU kernel can boot with architectures different from
+  On macOS 10.7 and earlier, the XNU kernel can boot with architectures different from
   the usual \texttt{x86\_64}. This setting will use the specified architecture to boot
   macOS when it is supported by the macOS and the configuration:
 
   \begin{itemize}
     \tightlist
-    \item \texttt{Auto} --- Choose the preferred architecture automatically.
     \item \texttt{i386} --- Use \texttt{i386} (32-bit) kernel when available.
     \item \texttt{i386-user32} --- Use \texttt{i386} (32-bit) kernel when available
       and force the use of 32-bit userspace on 64-bit capable processors if supported
-      by the operating system. On macOS 64-bit capable processors are assumed to
-      support \texttt{SSSE3}. This is not the case for older 64-bit capable Pentium
-      processors, which cause some applications to crash on macOS~10.6. This behaviour
-      corresponds to \texttt{-legacy} kernel boot argument. This option is unavailable
-      for 10.4 and 10.5 when running on 64-bit firmware due to an uninitialised 64-bit
-      segment in the XNU kernel, which causes AppleEFIRuntime to incorrectly execute
-      64-bit code as 16-bit code.
+      by the operating system.
+      \begin{itemize}
+        \tightlist
+        \item On macOS, 64-bit capable processors are assumed to support \texttt{SSSE3}.
+          This is not the case for older 64-bit capable Pentium processors, which cause some
+          applications to crash on macOS~10.6. This behaviour corresponds with the \texttt{-legacy}
```

Comment:
it is not really with. Using `i386-user32` here does the same thing `-legacy` boot argument does. The change in the wording might make it less obvious.